build.sbt

```diff
@@ -37,7 +37,7 @@ scalaVersion := "2.12.10"
 val sparkVersion    = "3.3.2"
 val airframeVersion = "23.6.1"
 val deltaVersion    = "2.2.0"
-version := s"1.1.6-spark${sparkVersion}-scala${scalaVersion.value}"
```

Comment:
do we really want to merge this version?

Reply:
Restored version.